chapters/ch25.md

```diff
@@ -1,31 +1,32 @@
 # Techniques for reducing avoidable bias
 ->
-
+# Các kĩ thuật để giảm độ chệch tránh được
 
 If your learning algorithm suffers from high avoidable bias, you might try the following techniques:
 ->
-
+Nếu thuật toán học của bạn gặp vấn đề với độ chệch tránh khỏi lớn, bạn có thể thử những kĩ thuật sau:
```

Comment:
> độ chệch tránh khỏi

ở tiêu đề là "độ chệch tránh được"

Reply:
@naml3i bạn vẫn chưa "resolve" comment này nhưng đã "marked as resolved".

Mình thấy comment của @william-vu khá hợp lý. Nếu bạn không đồng ý thì nên phản biện lại.

Reply:
hình như bạn ấy resolve rồi mà bác. Em có xem bản mới nhất bạn ấy up lên. Bác double check hộ em.

Reply:
e vừa check lại, bạn ấy mới resolve là tiêu đề, chưa sửa cái ở bên dưới.

Reply:
-> a unresolved lại rồi nhé.

Reply:
- Đã sửa. (Thực ra mình đã sửa hết theo suggestion của @william-vu mà commit nhầm, khi làm lại lại sót :D).

- Có một điểm mình chưa thấy ổn lắm là cả cụm "high avoidable bias", đọc tiếng Anh thì không sao (vì avoidable là một từ có mức compound rất lớn), khi sang tiếng Việt nó bị dàn ra quá lại kèm với từ "lớn" sau đó: "độ chệch có thể tránh được lớn", nên mình đề xuất để nghiêng cụm "avoidable bias" để có thể nhận định rõ "độ chệch có thể tránh được" là một khái niệm, "lớn" là tính từ bổ sung: "nếu thuật toán học của bạn gặp vấn đề với _độ chệch có thể tránh khỏi_ lớn (...)"

(Cái italic này không có trong bản gốc, nhưng cũng là một kĩ thuật thường gặp để 1 cụm dài nó stands out và đi với nhau nguyên vẹn, thay vì dùng ngoặc kép hay gạch nối. Mọi người cho ý kiến. Nếu ở các chương trước người đọc đã gặp nhiều khái niệm này thì chắc không cần phải in nghiêng).